### PR TITLE
[IOTDB-309] change docker.

### PR DESCRIPTION
**Pull request by Genius-pig**
https://issues.apache.org/jira/browse/IOTDB-309

### CODE REVIEW

**Comment by jixuan1989** [comment]
Hi, I am considering how to do this better.
1. we need to know how to upload the docker image to apache docker hub.
2. I find that if we compile the source code locally and just upload the bin files to the docker, then the image size can reduce to ~80MB.

**Comment by Genius-pig** [comment]
> Hi, I am considering how to do this better.
> 
> 1. we need to know how to upload the docker image to apache docker hub.
> 2. I find that if we compile the source code locally and just upload the bin files to the docker, then the image size can reduce to ~80MB.

+1, users can directly use `docker pull iotdb-version`.

**Comment by Genius-pig** [comment]
According to this issue, https://issues.apache.org/jira/browse/IOTDB-32.
We should also add Grafana service bin to image.

File: docker/Dockerfile [hunk 1]
```diff
@@ -28,22 +28,21 @@ RUN apt update \
...
-  && apt autoremove -y \
-  && apt purge --auto-remove -y \
-  && apt clean -y 
+  && apt-get remove wget maven unzip thrift-compiler -y \
+  && apt-get autoremove -y \
+  && apt-get purge --auto-remove -y \
+  && apt-get clean -y
 ENV JAVA_HOME  "/jdk-11.0.2"
 ENV PATH "$JAVA_HOME/bin:$PATH"
 EXPOSE 6667
```
Comment (jixuan1989) [inline docker/Dockerfile]: and we need to 
EXPOSE 31999

**Comment by Genius-pig** [comment]
I removed thrift and It build successfully in my computer. Can you try again? @jixuan1989

**Comment by qiaojialin** [comment]
https://github.com/apache/incubator-iotdb/pull/823

**Pull request closed without merging**